src/devtools/client/debugger/src/components/SecondaryPanes/Breakpoints/BreakpointNavigation.css

```diff
@@ -94,10 +94,3 @@
   stroke: var(--replaying-paused-marker-stroke);
 }
 
-.breakpoint-navigation-timeline-pause-marker {
```

Comment:
Hmm, this seems to delete the pause line, which we should probably evaluate in the context of another PR

Reply:
True. Fixed, thanks!